### ISSUE TITLE
k-fold cross validation

### ISSUE THREAD

**Issue by luna9722** [qubvel/segmentation_models#470]
Hello everyone,
Do anybody know how to do k fold cross validation using the notbook provided (camvid)? My dataset is too small (40 images for training and 10 images for test) I tried data augmentation and got 2400 images for taining, 600 for validation and 600 for test, but whatever I do (regularisatio, trying different data augmentation techniques) my model overfit and doesn't give good results with the testing set.
My results:
![image](https://user-images.githubusercontent.com/48836506/119978194-3e48b100-bfb1-11eb-8c32-9730ffaab077.png)

 and here some samples of the prediction on the test set:
![image](https://user-images.githubusercontent.com/48836506/119978341-6df7b900-bfb1-11eb-9db8-550a1c7127d1.png)
![image](https://user-images.githubusercontent.com/48836506/119978558-b1eabe00-bfb1-11eb-827d-1cca9c49727b.png)
Please, I will be grateful if you can help me solve this matter and try k fold cross validation. Thank you in advance.
